pentaho-database-gwt/src/org/pentaho/ui/database/event/DataHandler.java

```diff
@@ -1131,11 +1131,28 @@ public void restoreDefaults() {
         }
         item.getRow().addCellText( 2, defaultValue );
         item.getRow().addCellText( 0, "false" );
+        if ( "validationQuery".equals( parameterName ) ) {
+          item.getRow().setParentTreeItem( item );
+          item.getRow().getCell( 0 ).setTreeRowParent( findRow( "testOnBorrow" ) );
```

Comment:
Just for the sanity, shouldn't we check if findRow != null before we setTreeRowParent?